docs/en/sql-reference/functions/date-time-functions.md

```diff
@@ -435,6 +435,166 @@ Result:
 
 -   [toStartOfInterval](#tostartofintervaltime-or-data-interval-x-unit-time-zone)
 
+## date\_add {#date_add}
+
+Adds specified date/time interval to the provided date.
+
+**Syntax** 
+
+``` sql
+date_add(unit, value, date)
+```
+
+Aliases: `dateAdd`, `DATE_ADD`. 
+
+**Parameters**
+
+-   `unit` - The unit of time - [String](../syntax.md#syntax-string-literal).
```

Comment:
only it's not String

'second' -- String
second -- Literal ?

Reply:
yes, it's a `string literal`. Made the change.

Reply:
I think it's the other way around:
'second' - string literal: https://github.com/ClickHouse/ClickHouse/blob/master/docs/en/sql-reference/syntax.md#string-syntax-string-literal. Example: `HOUR` `MINUTE` etc.
second - string: https://github.com/ClickHouse/ClickHouse/blob/master/docs/en/sql-reference/data-types/string.md#string-string. In this case `HOUR`, `MINUTE` etc.

For all these examples, it should actually be String.